Resources/public/js/views/ez-contenteditview.js

```diff
@@ -279,7 +265,18 @@ YUI.add('ez-contenteditview', function (Y) {
              * @required
              */
             owner: {
-                value: {}
+                writeOnce: "initOnly",
+            },
+
+            /**
+             * The config of the content being edited
```

Comment:
no! Rather: "A configuration object coming from the application route configuration"